lib/nailinda/accounts/accounts.ex

```diff
@@ -73,4 +74,66 @@ defmodule Nailinda.Accounts do
   def change_user(%User{} = user) do
     User.changeset(user, %{})
   end
+
+  def create_patient(attrs) do
+    %Patient{}
+    |> Patient.changeset(attrs)
+    |> Repo.insert()
+  end
+
+  def update_patient(patient, attrs) do
+    patient
+    |> Patient.changeset(attrs)
+    |> Repo.update()
+  end
+
+  def get_all_patients do
+    Patient
+    |> Repo.all()
+  end
+
+  def get_patient_by_id(id) do
+    Patient
+    |> Repo.get(id)
+  end
+
+  def delete_patient(%Patient{} = patient) do
+    Repo.delete(patient)
+  end
+
+  def create_doctor(attrs) do
+    %Doctor{}
+    |> Doctor.changeset(attrs)
+    |> Repo.insert()
+  end
+
+  def get_doctor_by_id(id) do
+    Doctor
+    |> Repo.get!(id)
```

Comment:
if this repo function has a `!` then the main function should also contain it i.e `get_doctor_by_id!` to communicate to us that the function will throw an error if no result is found

Reply:
resolved by https://github.com/Podiihq/nailinda/issues/103